tests/test-naughty-legacy.lua

```diff
@@ -698,12 +698,12 @@ table.insert(steps, function()
     assert(n.text    == "baz")
 
     -- Test the ID system
-    n.id = 1337
-    assert(n.id == 1337)
-    assert(naughty.getById(1337) == n)
-    assert(naughty.get_by_id(1337) == n)
-    assert(naughty.getById(42) ~= n)
-    assert(naughty.get_by_id(42) ~= n)
+--     n.id = 1337
+--     assert(n.id == 1337)
+--     assert(naughty.getById(1337) == n)
+--     assert(naughty.get_by_id(1337) == n)
+--     assert(naughty.getById(42) ~= n)
+--     assert(naughty.get_by_id(42) ~= n)
```

Comment:
Use `set_id` for this test maybe?

Also could test the assertion error with setting it directly here.

Reply:
> Use set_id for this test maybe?

The property <-> accessor API is tested in thousands of places, it would be overkill.

> Also could test the assertion error with setting it directly here.

Done